src/models/sighting/useDeleteSighting.js

```diff
@@ -7,14 +7,25 @@ export default function useDeleteSighting() {
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState(null);
   const [success, setSuccess] = useState(false);
+  const [vulnerableIndividual, setVulnerableIndividual] = useState(
+    null,
+  );
 
-  const deleteSighting = async sightingId => {
+  const deleteSighting = async (
+    sightingId,
+    forceDeleteIndividual = false,
```

Comment:
What do you think about using a config object here instead for the second argument?

Reply:
Could you elaborate?

Reply:
something like
```js
const deleteSighting = async (sightingId, config = {}) {
  const { forceDeleteIndividual = false } = config;
  //...
}
```
which is similar to https://github.com/WildMeOrg/codex-frontend/blob/d8a689389916b96156f9b4858a2311cabffc4716/src/models/sighting/useSighting.js#L4

It is more verbose than what you have though.